src/js/content.js

```diff
@@ -34,6 +34,32 @@ var interceptPage = () => {
     document.body.insertBefore(newDiv, null); 
 }
 
+
+class UrlSettings {
```

Comment:
you can create a new file for this to make it cleaner, and then just include this as a content script in the manifest file. Scripts are loaded in in the order they are listed in the manifest file arrays, so it would have to appear before `content.js`